client/middleware/graphQL.js

```diff
@@ -0,0 +1,41 @@
+import {graphql} from 'graphql';
+
+var root = {
```

Comment:
to be replaced with the call to the backend

Reply:
I can see we are using `CREATE_CHARACTER`, surely that will be of type `mutation` rather than 'query' as we are creating a new record or are just reading for now?

Reply:
just reading, sorry names of functions are wrong atm there

Reply:
and i havent looked into mutation, this is just the very basics of adding graph ql